src/utils/Util.ts

```diff
@@ -0,0 +1,39 @@
+import mainApi from "../apis/mainApi";
+import { ERROR, SUCCESS } from "../shared/Constants";
+
+const convertTime = (time: string) => {
+  const timeSplit = time.split(":");
+  const hour = Number(timeSplit[0]);
+  const minute = Number(timeSplit[1]);
+  return hour * 60 + minute;
+};
+
+export const isValidTime = (withTime: string, expireTime: string) => {
+  const currentTime = new Date(Date.now() - new Date().getTimezoneOffset() * 60000)
+    .toISOString()
+    .slice(11, 16);
+  const convertCurrentTime = convertTime(currentTime);
+  const convertWithTime = convertTime(withTime);
+  const convertExpireTime = convertTime(expireTime);
+  if (convertCurrentTime >= convertWithTime) {
+    alert(`${ERROR.CREATE.EARLYWITHTIME} ${currentTime}`);
+    return false;
+  } else if (convertWithTime <= convertExpireTime) {
+    alert(`${ERROR.CREATE.LATEEXPIRETIME}`);
+    return false;
+  } else if (convertCurrentTime >= convertExpireTime) {
+    alert(`${ERROR.CREATE.EARLYEXPIRETIME}`);
+    return false;
+  } else return true;
```

Comment:
```suggestion
  if (convertCurrentTime >= convertWithTime) {
    alert(`${ERROR.CREATE.EARLYWITHTIME} ${currentTime}`);
    return false;
  }  
  if (convertWithTime <= convertExpireTime) {
    alert(`${ERROR.CREATE.LATEEXPIRETIME}`);
    return false;
  } 
  if (convertCurrentTime >= convertExpireTime) {
    alert(`${ERROR.CREATE.EARLYEXPIRETIME}`);
    return false;
  } 
  return true;
```

if else if를 사용하면서 depth를 늘리는거 보다 if return으로 작성하는 것이 코드 읽는데 훨신 편합니다.
https://velog.io/@jangws/9.-Else-if%EC%99%80-Else%EB%A5%BC-%ED%94%BC%ED%95%98%EA%B3%A0-Early-Return%ED%95%98%EC%9E%90

Reply:
수정하였습니다. [5187df7](https://github.com/By-hoon/cono-with-me-front/pull/35/commits/5187df7a9c2c050474f8d5a613971ce61fd4a252)